scripts/jenkins/cloud/ansible/roles/cloud_generator/templates/mkcloud.config.j2

```diff
@@ -42,11 +42,11 @@ export TESTHEAD=
 
 {% if upgrade_cloudsource.startswith("stagingcloud") %}
 export want_cloud{{ upgrade_cloudsource[-1] }}_iso_url={{ clouddata_server }}/repos/x86_64/
-export want_cloud{{ upgrade_cloudsource[-1] }}_iso={{ cloud_iso_name }}-devel-staging.iso
+export want_cloud{{ upgrade_cloudsource[-1] }}_iso=SUSE-OpenStack-Cloud-Crowbar-{{ upgrade_cloudsource[-1] }}-devel-staging.iso
 {% endif %}
 {% if upgrade_cloudsource.startswith("develcloud") %}
 export want_cloud{{ upgrade_cloudsource[-1] }}_iso_url={{ clouddata_server }}/repos/x86_64/
-export want_cloud{{ upgrade_cloudsource[-1] }}_iso={{ cloud_iso_name }}-devel.iso
```

Comment:
It would be better to add a `upgrade_cloud_iso_name` variable next to https://github.com/SUSE-Cloud/automation/blob/master/scripts/jenkins/cloud/ansible/roles/cloud_generator/vars/product/crowbar.yml#L18 and use that here.

Reply:
OK, good point